README.md

```diff
@@ -184,6 +184,23 @@ spec:
     # Array of strings (format is either with prefix '--key=value' or just 'value')
     # args: []
 
+    ## Audit Config
+    # https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#audit-policy
+    # Define basic arguments for auditing log files. If needed, more complex flags
+    # can be set via 'Args' field manually.
+    # auditConfig:
+    #   logFormat: "json"
+    #   LogOutputVolumeClaim: ""
```

Comment:
```suggestion
    #   logOutputVolumeClaim: ""
```

Reply:
would be nice to document a behavior here, where the log is stored if this option si specifed and what should be actually specified here?